addons/stock_picking_batch/wizard/stock_package_destination.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class ChooseDestinationLocation(models.TransientModel):
+    _inherit = "stock.package.destination"
+
+    @api.depends('picking_id')
```

Comment:
@api.depends('picking_id') is useless. Overwrite of compute keep depends

Reply:
Good to know, removed it.